index.html

```diff
@@ -57,14 +58,299 @@ <h3 class="activity">I am a Full Stack Developer</h2>
             <img src="./images/victory-icon.png" class="icon victory" alt="victory icon">
           </a>
         </li>
+        <li>
+          <a href="#">
+            <img src="./images/twitter-icon.png" class="icon twitter" alt="twitter icon">
+          </a>
+        </li>
         <li>
           <a href="#">
             <img src="./images/m-icon.png" class="icon gmail" alt="gmail icon">
           </a>
         </li>
       </ul>
-      
     </div>
   </section>
+  
+  <section class="mid-section">
+    <div class="item mrw">
+      <p>My Recent Works</p>
+    </div>
```

Comment:
- [ ] I think you should consider using an ```<hx>``` tag instead of a p tag here since ```My Recent Works``` is a heading for the section. This improves semantics 👍 . Example, line 77 could be replaced with ```<h3>My Recent Works</h3>```